servlet/src/main/scala/org/http4s/servlet/syntax/ServletContextSyntax.scala

```diff
@@ -38,23 +39,43 @@ final class ServletContextOps private[syntax] (val self: ServletContext) extends
     *
     * Assumes non-blocking servlet IO is available, and thus requires at least Servlet 3.1.
     */
-  def mountService[F[_]: Async](
+  @deprecated("Use mountRoutes instead", "1.0.0-M32")
+  private[servlet] def mountService[F[_]: Async](
```

Comment:
I think we shouldn't make this private

Reply:
Yeah, I wasn't sure about this one but it is a good practice to make deprecated methods private. It will break source compatibility but it is not an issue for binary compatibility.

Reply:
I'm 👎 on breaking source-compatibility in a patch release. It doesn't follow our versioning scheme; to break source-compatibility requires a minor version bump i.e. http4s 0.24.x.
> you MUST increment the MINOR version if backward source compatibility is broken

https://docs.scala-lang.org/overviews/core/binary-compatibility-for-library-authors.html#recommended-versioning-scheme

Reply:
We have broken source compatibility in patches before, but it should never be done without urgent reason.  I agree with deprecation but not privatization.